octree_web_viewer/src/bin/points_web_viewer.rs

```diff
@@ -73,7 +73,10 @@ fn main() {
     let octree_directory = PathBuf::from(matches.value_of("octree_directory").unwrap());
 
     let octree = {
-        let octree = match octree::OnDiskOctree::new(octree_directory) {
+        let data_provider = octree::OnDiskOctreeDataProvider {
```

Comment:
How about a more user-friendly interface: Octree::from_directory() that hides the data provider?

Reply:
Yes, good idea, I just added two free functions `octree_from_directory` and `octree_from_address`.